src/components/drop-down/drop-down.tsx

```diff
@@ -10,12 +11,34 @@ export interface DropDownInputProps {
 }
 
 export const DropDownInput: React.SFC<DropDownInputProps> = SBStateless(props => {
-  return (
-      <div data-automation-id="DROP_DOWN_INPUT" onClick={props.onClick} className="drop-down-input">
-          <span className="label">{props.selectedItem ? props.selectedItem.label : 'Default Text'}</span>
-          <CaretDown className="caret" />
-      </div>
-  );
+    return (
+        <div data-automation-id="DROP_DOWN_INPUT" onClick={props.onClick} className="drop-down-input">
+            <span className="label">{props.selectedItem ? props.selectedItem.label : 'Default Text'}</span>
+            <CaretDown className="caret" />
+        </div>
+    );
+}, style);
+
+export interface DropDownListProps {
+    open: boolean;
+    items?: DropDownItem[];
```

Comment:
Right now `items` and `selectedItem` have the same type, which is absolutely reasonable, but we're not going to be able to make this work with `SelectionList`, because it uses two different types for `dataSource` and for `value`/`onChange`.

There are two options right now.
1) To convert any data source to an array of strings, which is what you're doing here:
```
dataSource={props.items!.map((item: DropDownItem) => item.label)}
```

2) To make `DropDownListProps` extend `OptionList` from 'selection-list.tsx'. DropDown would simply pass dataSource from its props to SelectionList without giving a damn about its contents. And this would allow to a use custom renderer for the items.

I'm looking at the [spec](https://github.com/wix/stylable-components/pull/103/files) right now, and it basically says that dropdown should extend OptionList, and have the `value` and `onChange` props, and pass its children to SelectionList.